Makefile

```diff
@@ -17,10 +16,10 @@ DOCKER_BUILD_OPTIONS =
 
 GIT_COMMIT=$(shell git rev-parse --short HEAD)
 
+export DOCKER_BUILDKIT = 1
+
 nginx-ingress:
-ifeq ($(BUILD_IN_CONTAINER),1)
```

Comment:
similar thing should be done to the test stage - run it inside the builder container, not here